student.rb

```diff
@@ -0,0 +1,16 @@
+require 'Person'
+
+# student.rb
+class Student < Person
+  def initialize(classroom, name, age, parent_permission)
+    super(name, age, parent_permission)
```

Comment:
- [ ] This highlighted code should look like the example code below :+1: , you are missing the default values for the name and parent_permission 
- code snippet
```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)

```